app/controllers/api/v3/scenarios_controller.rb

```diff
@@ -88,7 +88,7 @@ def create
 
         @scenario = Scenario.new
 
-        if scaler_attributes && ! attrs[:descale]
+        if scaler_attributes && ! attrs[:descale] && ! Area.derived?(attrs.fetch(:area_code))
```

Comment:
* Any chance of a test for this? [There are tests for other two conditions](https://github.com/quintel/etengine/blob/470256764853fa338a13543e272de324bd46610b/spec/requests/api/v3/create_scenario_spec.rb#L217-L320) (scaler attributes being provided and/or the `descale` option).

* It seems like this would make impossible to (further) scale a derived region into something smaller. Would this not be useful to have in the event that you want to scale a derived scenario for use in Moses? For example, perhaps to scale Ameland into a sub-region representing only a couple of streets?

  I guess the problem is that this means having two sets of scaling settings (one for the region, one for the scenario), and that might be a pain to manage. I guess the expectation is that one day we'll do away with the old method of scaling entirely, but that seems quite far off... @ChaelKruip does it seem okay to you to *not* be able to (re-)scale derived scenarios? I can imagine it being annoying to some users.